feature-libs/cart/import-export/components/import-to-cart/import-to-cart.service.ts

```diff
@@ -55,12 +63,12 @@ export class ImportToCartService {
               });
               this.savedCartService.loadSavedCarts();
             }),
-            // TODO: #13392
-            // delayWhen((cartId: string) =>
-            //   this.savedCartService
-            //     .isStable(cartId)
-            //     .pipe(filter((stable: boolean) => stable))
-            // ),
+            observeOn(queueScheduler),
+            delayWhen(() =>
+              this.savedCartService
+                .getSaveCartProcessLoading()
+                .pipe(filter((loading) => !loading))
+            ),
```

Comment:
The SaveCart action doesn't set the loading flag on the cart state (unfortunately), but instead it sets the loading flag on the separate process. That's why we observe it.

IMHO it would be more consistent if SaveCart set the loading flag on the specific cart's state. But that would be a breaking change.